queries/rust/injections.scm

```diff
@@ -5,3 +5,5 @@
   (macro_rule
     left: (token_tree_pattern) @rust
     right: (token_tree) @rust))
+
+(comment) @comment
```

Comment:
From what I remember rust also has `block_comment` which should be highlighted too?